content/contact/contents.lr

```diff
@@ -13,7 +13,7 @@ body:
 
 If you have a general question about BeeWare usage, we have a number of `community communication channels </community/getting-help/>`__ where you can ask for assistance.
 
-If you need advice on how to contribute, you have a questions about the management of the project, anyone on the `Core Team </community/team>`__ will be able to help you.
+If you need advice on how to contribute, you have questions about the management of the project, anyone on the `Core Team </community/team>`__ will be able to help you.
```

Comment:
Thanks for looking at this! Could you change it to be `...contribute, or you have questions...`? There should be an `or` there.

Reply:
Oh, sure! I am glad if I can make some contribution to this project